tsup.config.js

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from "tsup";
+
+export default defineConfig({
+  entry: ["src/sdk/index.ts"],
+  format: ["cjs", "esm"],
+  outDir: "dist",
+  splitting: false,
+  sourcemap: true,
```

Comment:
Should not be used on production: https://tsup.egoist.dev/#generate-sourcemap-file